src/extension.ts

```diff
@@ -51,6 +52,9 @@ export async function activateInternal(
         ext.fileEditor = new FileEditor();
         context.subscriptions.push(ext.fileEditor);
 
+        const uriHandler = new ImportTrialAppUriHandler();
```

Comment:
No reason to declare a variable here since you don't use it for anything else (except to push into the disposables array)